src/components/UserInfo.js

```diff
@@ -11,8 +12,13 @@ export default class UserInfo {
     };
   }
 
-  setUserInfo({ name, occupation }) {
+  setUserInfo({ name, about, avatar }) {
     this._name.textContent = name;
-    this._occupation.textContent = occupation;
+    this._occupation.textContent = about;
+    this._avatar.src = avatar;
```

Comment:
`    this._avatar.src = avatar;
`


You have method `changeUserPhoto` for that. Please, use it here not to duplicate the code